demos/mqtt/mqtt_demo_basic_tls/mqtt_demo_basic_tls.c

```diff
@@ -184,18 +431,40 @@ static SSL * tlsSetup( int tcpSocket )
 
         /* OpenSSL does not provide a single function for reading and loading certificates
          * from files into stores, so the file API must be called. */
-        pRootCaFile = fopen( SERVER_CERT, "r" );
+        pRootCaFile = fopen( SERVER_CERT_PATH, "r" );
 
         if( pRootCaFile != NULL )
         {
             pRootCa = PEM_read_X509( pRootCaFile, NULL, NULL, NULL );
         }
+        else
+        {
+            LogError( ( "Unable to find the certificate file in the path"
+                        " provided by SERVER_CERT_PATH(%.*s).",
+                        SERVER_CERT_PATH_LEN,
+                        SERVER_CERT_PATH ) );
```

Comment:
Is it preferable to use %.*s over %s?

Reply:
%s relies on internally calculating strlen, %.*s comes with a benefit that we can pass the already known string length. Also %s relies on the presence of a null character to the end. In this above instance it shouldn't make a difference.

Reply:
Good to know. Thank you.